test/create_test_environment.sh

```diff
@@ -73,7 +79,7 @@ gcloud beta billing projects link $GOOGLE_CLOUD_PROJECT --billing-account=$GOOGL
 
 gcloud iam service-accounts create $GCS_SA \
     --project=$GOOGLE_CLOUD_PROJECT \
-    --display-name $GCS_SA
+    --display-name $GCS_SA || true
```

Comment:
I would prefer to not use `|| true` here but check for the specific error / error code.
If not possible, I would just abort, the user can do it manually if she has a special case